README.md

```diff
@@ -31,27 +29,37 @@ You can choose your own path among the following levels:
 
 **TASKS**
 + *All of the base requirements*
-+ Create and implement (using HTML, CSS and JS) a UI for listing the [short list of systems](https://github.com/SUSE/frontend-challenge/blob/master/systems-short-list.json) systems with a basic design
++ Use a JavaScript framework ([ReactJS](https://reactjs.org/) is highly recommended)
++ Use a CSS Framework ([Bootstrap](https://getbootstrap.com/) is preferred)
++ Make use of the [GitHub API](https://developer.github.com/) (any version) to connect to the [SaltStack/Salt](https://github.com/saltstack/salt) GitHub repository to fetch the data
++ Create and implement a UI for reporting statistics of the github repository:
+  + open Projects
+  + open/closed Pull Requests
+  + Labels
+  + open/closed Issues
+  + ..
 
 ## Intermediate Level
 
 **TASKS**
 + *All of the base and entry requirements*
-+ Use a JavaScript framework (ReactJS is preferred)
-+ Use a CSS Framework (Bootstrap is preferred)
-+ Use LESS for custom CSS
++ Use [Yarn](https://yarnpkg.com/) to set up the project
 + Cross browser support (or graceful degradation)
-+ Responsive design and implementation
```

Comment:
Why are we considering responsive design only on the advanced level?

Reply:
Well, on one hand responsive design is not a big deal form the technical POV, but it is also something that can annoy the implementation without focusing on other tasks. We could move to the `intermediate` maybe? Basically it is just more in line with our current situation: we don't have a responsive product interface. Fine to move to intermediate?

Reply:
Moved to `intermediate` for now.